main.css

```diff
@@ -0,0 +1,84 @@
+body {
+    background-image: url("https://raw.githubusercontent.com/Yjohn/WebDeveloperTest/master/assets/background.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    text-decoration: none;
+}
+
+#wrap {
+    width: 100%;
+    color: white;
+    font-family: sans-serif;
+    font-weight: 100;
+}
+
+.main {
+    width: 75%;
+    padding: 2px 30px;
+    font-weight: 100;
+}
+
+.nav-item {
+    display: inline;
+}
+
+.nav-item a {
+    text-decoration: none;
+    color: white;
+    padding: 0.1em 2em;
+    border: 0.1px solid;
+    border-right: white;
+    border-bottom: black;
+    background-image: linear-gradient( white, black 20%);
+}
+
+h1 {
+    font-family: 'Apple-Chancery', cursive;
+    font-weight: 100;
+    font-style: italic;
+    font-size: 60px;
+}
+
+h2 {
+    color: #CB3541;
+    font-weight: 100;
+}
+
+h3 {
+    font-weight: 100;
+}
+
+span {
+    color: #CB3541;
+    content: "\203A";
+}
+
+.ourYellow {
```

Comment:
Try avoid naming classes like this (with the colour 'yellow' in the class name) — what would happen if the client asks to change the colour from yellow to, for example, red?